azure_arc_servers_jumpstart/docs/aws_scale_ansible.md

```diff
@@ -0,0 +1,184 @@
+# Onboard AWS servers to Azure Arc at scale using Ansible
+
+The following README will guide you on how to automatically onboard servers to Azure Arc by using [Ansible](https://www.ansible.com/). This guide assumes that you have a basic understanding of Ansible. A basic Ansible playbook and configuration is provided that uses the [amazon.aws.aws_ec2](https://docs.ansible.com/ansible/latest/collections/amazon/aws/aws_ec2_inventory.html) plugin for dynamic loading of EC2 server inventory. 
+
+This guide can be used even if you do not already have an existing Ansible test environment. The guide includes a Terraform plan that will create a sample AWS EC2 server inventory comprised of four (4) Windows Server 2019 servers and four (4) Ubuntu servers along with a basic CentOS 7 Ansible control server with a simple configuration.
+
+***Warning***: *The provided Ansible sample workbook uses WinRM with password authentication and HTTP to configure Windows-based servers. This is not advisable for production environments. If you are planning to use Ansible with Windows hosts in a production environment then you should use WinRM over HTTPS with a certificate.*
+
+# Prerequisites
+
+* Clone this repo
+
+    ```terminal
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+    
+* [Install or update Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). **Azure CLI should be running version 2.7** or later. Use ```az --version``` to check your current installed version.
+
+* [Generate SSH Key](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) (or use existing ssh key) 
+
+* [Create free AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)
+
+* [Install Terraform >=0.13](https://learn.hashicorp.com/terraform/getting-started/install.html)
+
+## Create Azure Service Principal (SP)   
+
+* To connect the AWS virtual machine to Azure Arc, an Azure Service Principal assigned with the "Contributor" role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)). 
+
+    ```bash
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```az ad sp create-for-rbac -n "http://AzureArcAWS" --role contributor```
```

Comment:
```suggestion
    ```console
    az ad sp create-for-rbac -n "http://AzureArcAWS" --role contributor
```
```